tests/unit/other.tcl

```diff
@@ -374,3 +374,20 @@ start_server {tags {"other external:skip"}} {
     }
 }
 
+test {hash-seed configuration works as expected} {
+    proc make_hash {seed} {
+        start_server [list overrides [list hash-seed $seed]] {
```

Comment:
i would like to add a test that checks the random see (when `hash-seed` is not used).
and make sure the main dict and some other dict are indeed random.

Reply:
Makes sense, that would be a serious regression.